src/content/add-to-app/ios/project-setup.md

```diff
@@ -393,99 +400,105 @@ as described in Option B.
 
 ## Local Network Privacy Permissions
 
-On iOS 14 and higher, enable the Dart multicast DNS
-service in the Debug version of your app
-to add [debugging functionalities such as hot-reload and
-DevTools][] using `flutter attach`.
+On iOS 14 and later, enable the Dart multicast DNS service in the
+Debug version of your app to add
+[debugging functionalities such as hot-reload and DevTools][]
+using `flutter attach`.
 
 :::warning
 This service must not be enabled in the **Release**
 version of your app, or you might experience App Store rejections.
 :::
 
-One way to do this is to maintain a separate copy of your app's Info.plist per
-build configuration. The following instructions assume
-the default **Debug** and **Release**.
+Maintain a separate copy of your app's Info.plist per build configuration.
+The following instructions assume the default **Debug** and **Release**.
 Adjust the names as needed depending on your app's build configurations.
 
-<ol>
-<li>
+1. Rename your app's **Info.plist** to **Info-Debug.plist**.
```

Comment:
These instructions are hard to follow if you have a new Swift app as they don't have a `Info.plist` file and Xcode's add file dialog doesn't have a `.plist` option.

Consider adding a note like:

```
SwiftUI projects start without an `Info.plist` file. You can create one by going to your target's `Info` > `Custom iOS Target Properties` and editing a property.
```